scripts/models/TownHall.js

```diff
@@ -128,12 +128,43 @@
     });
   };
 
-  TownHall.lookupReps = function (zip) {
-    var representativePromise = $.ajax({
-      url: 'https://congress.api.sunlightfoundation.com/legislators/locate?zip=' + zip,
-      dataType: 'jsonp'
+  TownHall.lookupReps = function (districts) {
+    return new Promise(function (resolve, reject) {
+      var state = districts[0].split('-')[0];
+      districts = districts.map(function(district) {
+        return parseInt(district.split('-')[1]);
+      });
+      districts.push(null);
+
+      // Sunlight allows you to filter by districts...but if you do then senators don't show up, so we pull them all and look
+      // for those with matching districts or no district at all.
+      $.ajax({
+        url: 'https://congress.api.sunlightfoundation.com/legislators?per_page=all&state=' + state,
```

Comment:
Sure I'll request an api key from them (guess they re-enabled them now?).

Want me to also take a quick look into seeing the feasiblity of using google civic's api for this while I'm at it?  If that's where we're moving over to then it makes sense to at least give it a quick check.